releasenotes/notes/logs-support-compression-for-http-endpoint-55828e4c5697cafe.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Support compression for logs HTTP endpoints
```

Comment:
I'd rather remove this release note as we don't want to advertise this feature yet.